.github/workflows/simple-pull.yaml

```
@@ -0,0 +1,46 @@
+  packages: write
+
+jobs:
+  check-conditions:
+    if: github.event_name == 'pull_request' && github.event.action != 'closed'
+    runs-on: ubuntu-latest
+    outputs:
+      should-proceed: ${{ steps.evaluate.outputs.should_proceed }}
+    steps:
+      - name: Apply labels
+        if: github.event_name == 'pull_request' && github.event.action != 'closed'
+        uses: actions/labeler@v4
```

actionlint (1.7.12)

[error] the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)